CHANGELOG.md

```diff
@@ -1,64 +1,54 @@
 # Changelog
 
-> **Tags:**
->
-> - :boom: [Breaking Change]
-> - :eyeglasses: [Spec Compliancy]
-> - :rocket: [New Feature]
-> - :bug: [Bug Fix]
-> - :memo: [Documentation]
-> - :house: [Internal]
-> - :nail_care: [Polish]
```

Comment:
Seems we will need to move this somewhere else - changesets is not smart enough to add changes below this. I'm sure there is a way but we can figure that out later unless you are sold on these being here 🙂

Reply:
Yeah nice to have so I think the benefits of auto-generating the changelogs outweighs the legend :)